events/story_cycles/wc_story_cycle_horde_invasion_events.txt

```diff
@@ -47,34 +47,22 @@ WCHOI.2 = {
 	}
```

Comment:
Too blurry. Also, I would fix Gender Stratification icon.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114469119-998b3380-9bfd-11eb-9771-eede7d1cf3ba.png)
![image](https://user-images.githubusercontent.com/46384992/114469358-06063280-9bfe-11eb-9da1-eab8fd42d004.png)
![image](https://user-images.githubusercontent.com/46384992/114469366-0acae680-9bfe-11eb-902c-45f878b7f8fb.png)
![image](https://user-images.githubusercontent.com/46384992/114469402-2209d400-9bfe-11eb-9969-10459363ba14.png)

</details>

Reply:
Resaved

Reply:
It's still blurry compared to the vanilla icons, but fine.